[core]/es_extended/server/functions.lua

```diff
@@ -259,6 +259,36 @@ function ESX.GetExtendedPlayers(key, val)
 	return xPlayers
 end
 
+function ESX.GetNumPlayers(key, val)
+    if(not key) then
```

Comment:
remove please () only add  if not key then

Reply:
ok